src/main/java/com/autotune/database/helper/DBHelpers.java

```diff
@@ -196,6 +197,51 @@ public static List<CreateExperimentAPIObject> convertExperimentEntryToCreateExpe
 
                 return createExperimentAPIObjects;
             }
+            public static List<UpdateResultsAPIObject> convertResultEntryToUpdateResultsAPIObject(List<KruizeResultsEntry> kruizeResultsEntries) throws JsonProcessingException {
+                ObjectMapper mapper = new ObjectMapper();
+                List<UpdateResultsAPIObject> updateResultsAPIObjects = new ArrayList<>();
+                for (KruizeResultsEntry kruizeResultsEntry : kruizeResultsEntries) {
+                    UpdateResultsAPIObject updateResultsAPIObject = new UpdateResultsAPIObject();
```

Comment:
Put try and catch inside for loop
so that if one result object is corrupted it should continue to load other results objects

Reply:
Done